packages/tasks/src/library-ui-elements.ts

```diff
@@ -541,6 +541,13 @@ transcriptions = asr_model.transcribe(["file.wav"])`,
 
 const mlAgents = (model: ModelData) => [`mlagents-load-from-hf --repo-id="${model.id}" --local-dir="./downloads"`];
 
+const mlx = (model: ModelData) => [
+	`pip install huggingface_hub hf_transfer
+
+export HF_HUB_ENABLE_HF_TRANSFER=1
```

Comment:
personally i would remove the HF_HUB_ENABLE_HF_TRANSFER line (it's a bit experimental + not always faster depending on machine)